tests/test_data/40aprons.com/fortyaprons.json

```
@@ -0,0 +1,110 @@
+{
+  "author": "Cheryl Malik",
+  "canonical_url": "https://40aprons.com/rosemary-raspberry-vodka-fizz/",
+  "site_name": "40 Aprons",
+  "host": "40aprons.com",
+  "language": "en-US",
+  "title": "Raspberry Cocktails with Rosemary",
+  "ingredients": [
+    "1 tablespoon rosemary needles",
+    "1 ½ cups frozen raspberries (defrosted)",
+    "juice of 1 lemon (approximately 2 tablespoons)",
+    "5 tablespoons sugar (¼ cup + 1 tablespoon)",
+    "¾ cup water",
+    "1 tablespoon cornstarch",
+    "1 ½ ounces vodka (or gin)",
+    "½ ounce St. Germaine elderflower liqueur",
+    "1 ½ ounces raspberry-rosemary syrup (made from ingredients above)",
+    "ice",
+    "1 tablespoon soda water",
+    "1 tablespoon champagne",
+    "ice",
+    "rosemary sprigs",
+    "raspberries"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1 tablespoon rosemary needles",
+        "1 ½ cups frozen raspberries (defrosted)",
+        "juice of 1 lemon (approximately 2 tablespoons)",
+        "5 tablespoons sugar (¼ cup + 1 tablespoon)",
+        "¾ cup water",
+        "1 tablespoon cornstarch"
+      ],
+      "purpose": "For the Raspberry-Rosemary Syrup (Makes Enough for 6-8 Cocktails)"
```

```suggestion
      "purpose": "For the Raspberry-Rosemary Syrup (Makes Enough for 6–8 Cocktails)"
```
Small character change here as well just from how the website portrays its information.